include/bluetooth/mesh/light_ctl_srv.rst

```diff
@@ -0,0 +1,128 @@
+.. _bt_mesh_light_ctl_srv_readme:
+
+Light CTL Server
+######################
+
+The Light Color Tunable Light (CTL) Server represents a single light on a mesh device.
+It should be instantiated in the light fixture node.
+
+The Light CTL Server adds the following new model instances in the composition data, in addition to the extended :ref:`bt_mesh_lightness_srv_readme` model:
+
+* Light CTL Server
+* Light CTL Setup Server
+
+These model instances share the states of Light CTL Server, but accept different messages.
+This allows for a fine-grained control of the access rights for the Light CTL states, as these model instances can be bound to different application keys:
+
+* The Light CTL Server provides write access to the Lightness, Temperature and Delta UV states for the user, in addition to read access to all the meta states.
+* The Light CTL Setup Server provides write access to the Default CTL state and Temperature Range meta states.
+  This allows the configurator devices to set up the Temperature range and the default CTL state.
+
+In addition to the extended Lightness Server model the Light CTL Server is also assosiated with an instance of the :ref:`bt_mesh_light_temp_srv_readme` model.
+This model instance must be located on the same device, where the element address of the Light Temperature Server must be greater than the address of the
+Light CTL Server.
```

Comment:
This is the most likely error users of this model are going to make, so I'd like to be very clear about this. Suggested change for this paragraph:

---

In addition to the extended Lightness Server model, the Light CTL Server also requires a :ref:`bt_mesh_light_temp_srv_readme` to be instantiated on a subsequent element. The Light Temperature Server should reference the :c:member:`bt_mesh_light_ctl_srv.temp_srv`.

Conventionally, the Light Temperature Server model is instantiated on the very next element, and the composition data looks like this:

```C
static struct bt_mesh_light_ctl_srv light_ctl_srv = BT_MESH_LIGHT_CTL_SRV_INIT(&light_ctl_handlers);

static struct bt_mesh_elem elements[] = {
	BT_MESH_ELEM(
		1,
		BT_MESH_MODEL_LIST(BT_MESH_MODEL_LIGHT_CTL_SRV(&light_ctl_srv)),
		BT_MESH_MODEL_NONE),
	BT_MESH_ELEM(
		2,
		BT_MESH_MODEL_LIST(BT_MESH_MODEL_LIGHT_TEMP_SRV(&light_ctl_srv.temp_srv)),
		BT_MESH_MODEL_NONE),
};
```

..warning::
   The Light CTL Server will verify that its internal Light Temperature Server is instantiated on a subsequent element on startup. If the Light Temperature Server is missing or instantiated on the same or a preceding element, the Bluetooth Mesh startup procedure will fail, and the device will not be responsive.